packages/web-components/fast-components/src/menu-item/menu-item.styles.ts

```diff
@@ -272,7 +273,7 @@ export const menuItemStyles: (
         bottom: 4px;
         border-radius: 999px;
         display: block;
-        background: ${neutralForegroundRest};
+        background: ${foregroundOnAccentRest};
```

Comment:
The `fill` setting doesn't work here since it's not svg, but we'll be fixing that soon, in line with the change I made in Fluent.